<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSimplePaintClass</class>
 <widget class="QMainWindow" name="QSimplePaintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>untitled - QSimplePaint[*]</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDeleteAll"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionEraser"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionRectFill"/>
    <addaction name="actionRectRound"/>
    <addaction name="actionRectRoundFill"/>
    <addaction name="actionCircle"/>
    <addaction name="actionCircleFill"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLine_Thickness">
     <property name="title">
      <string>Line Thickness</string>
     </property>
     <addaction name="actionLineWidth1"/>
     <addaction name="actionLineWidth2"/>
     <addaction name="actionLineWidth3"/>
     <addaction name="actionLineWidth4"/>
     <addaction name="actionLineWidth5"/>
    </widget>
    <widget class="QMenu" name="menuLine_Style">
     <property name="title">
      <string>Line Style</string>
     </property>
     <addaction name="actionSolidLine"/>
     <addaction name="actionDash"/>
     <addaction name="actionDot"/>
     <addaction name="actionDashDot"/>
     <addaction name="actionDashDotDot"/>
    </widget>
    <widget class="QMenu" name="menuBrush_Style">
     <property name="title">
      <string>Brush Style</string>
     </property>
     <addaction name="actionSolidBrush"/>
     <addaction name="actionExtremelyDense"/>
     <addaction name="actionVeryDense"/>
     <addaction name="actionSomewhatDense"/>
     <addaction name="actionHalfDense"/>
     <addaction name="actionSomewhatSparse"/>
     <addaction name="actionVerySparse"/>
     <addaction name="actionExtremelySparse"/>
     <addaction name="separator"/>
     <addaction name="actionHorizontalLines"/>
     <addaction name="actionVerticalLines"/>
     <addaction name="actionSquares"/>
     <addaction name="actionBackwardDiagonals"/>
     <addaction name="actionForwardDiagonals"/>
     <addaction name="actionCrossedDiagonals"/>
    </widget>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="actionBorderColor"/>
     <addaction name="actionBrushColor"/>
    </widget>
    <addaction name="menuColor"/>
    <addaction name="menuLine_Thickness"/>
    <addaction name="menuLine_Style"/>
    <addaction name="menuBrush_Style"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMainToolbar"/>
    <addaction name="actionPaintToolbar"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomOriginal"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionDeleteAll"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomOriginal"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286528461_picture_add.png</normaloff>:/QSimplePaint/Resources/1286528461_picture_add.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286528499_folder_picture.png</normaloff>:/QSimplePaint/Resources/1286528499_folder_picture.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/picture_save.png</normaloff>:/QSimplePaint/Resources/picture_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286994429_picture_saveas.png</normaloff>:/QSimplePaint/Resources/1286994429_picture_saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286994517_print_32.png</normaloff>:/QSimplePaint/Resources/1286994517_print_32.png</iconset>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286527884_Exit.png</normaloff>:/QSimplePaint/Resources/1286527884_Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286992397_picture_go1.png</normaloff>:/QSimplePaint/Resources/1286992397_picture_go1.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286528508_picture_go.png</normaloff>:/QSimplePaint/Resources/1286528508_picture_go.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDeleteAll">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286528428_picture_delete.png</normaloff>:/QSimplePaint/Resources/1286528428_picture_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/stock_draw-line.png</normaloff>:/QSimplePaint/Resources/stock_draw-line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286527722_stock_draw-rectangle-unfilled.png</normaloff>:/QSimplePaint/Resources/1286527722_stock_draw-rectangle-unfilled.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionRectFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286527727_stock_draw-rectangle.png</normaloff>:/QSimplePaint/Resources/1286527727_stock_draw-rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rect Fill</string>
   </property>
  </action>
  <action name="actionRectRound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/stock_draw-rounded-rectangle-unfilled.png</normaloff>:/QSimplePaint/Resources/stock_draw-rounded-rectangle-unfilled.png</iconset>
   </property>
   <property name="text">
    <string>Rect Round</string>
   </property>
  </action>
  <action name="actionRectRoundFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/stock_draw-rounded-rectangle.png</normaloff>:/QSimplePaint/Resources/stock_draw-rounded-rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rect Round Fill</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/stock_draw-ellipse-unfilled.png</normaloff>:/QSimplePaint/Resources/stock_draw-ellipse-unfilled.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionCircleFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/stock_draw-ellipse.png</normaloff>:/QSimplePaint/Resources/stock_draw-ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Circle Fill</string>
   </property>
  </action>
  <action name="actionLineWidth1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionLineWidth2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionLineWidth3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionLineWidth4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionLineWidth5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionSolidLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionDash">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dash</string>
   </property>
  </action>
  <action name="actionMainToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Main Toolbar</string>
   </property>
  </action>
  <action name="actionPaintToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paint Toolbar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286995465_report_picture.png</normaloff>:/QSimplePaint/Resources/1286995465_report_picture.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/zoomin.png</normaloff>:/QSimplePaint/Resources/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Alt++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/zoomout.png</normaloff>:/QSimplePaint/Resources/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Alt+-</string>
   </property>
  </action>
  <action name="actionZoomOriginal">
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/1286995080_Zoom.png</normaloff>:/QSimplePaint/Resources/1286995080_Zoom.png</iconset>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qsimplepaint.qrc">
     <normaloff>:/QSimplePaint/Resources/eraser_minus.png</normaloff>:/QSimplePaint/Resources/eraser_minus.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionDot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dot</string>
   </property>
  </action>
  <action name="actionDashDot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dash-Dot</string>
   </property>
  </action>
  <action name="actionDashDotDot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dash-Dot-Dot</string>
   </property>
  </action>
  <action name="actionSolidBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionExtremelyDense">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Extremely Dense</string>
   </property>
  </action>
  <action name="actionVeryDense">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Very Dense</string>
   </property>
  </action>
  <action name="actionSomewhatDense">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Somewhat Dense</string>
   </property>
  </action>
  <action name="actionHalfDense">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Half Dense</string>
   </property>
  </action>
  <action name="actionSomewhatSparse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Somewhat Sparse</string>
   </property>
  </action>
  <action name="actionVerySparse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Very Sparse</string>
   </property>
  </action>
  <action name="actionExtremelySparse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Extremely Sparse</string>
   </property>
  </action>
  <action name="actionHorizontalLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal Lines</string>
   </property>
  </action>
  <action name="actionVerticalLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical Lines</string>
   </property>
  </action>
  <action name="actionSquares">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Squares</string>
   </property>
  </action>
  <action name="actionForwardDiagonals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Forward Diagonals</string>
   </property>
  </action>
  <action name="actionBackwardDiagonals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backward Diagonals</string>
   </property>
  </action>
  <action name="actionCrossedDiagonals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Crossed Diagonals</string>
   </property>
  </action>
  <action name="actionBorderColor">
   <property name="text">
    <string>Border Color...</string>
   </property>
  </action>
  <action name="actionBrushColor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Brush Color...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qsimplepaint.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionMainToolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>mainToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QSimplePaintClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
